src/backend/app/app.py

```diff
@@ -1,16 +1,29 @@
 from flask import Flask, render_template, request, redirect, url_for, flash
 import sqlite3
```

Comment:
I think you removed all the existing imports while merging , cause I cant find RAG imports  and changed the database as well . I am rewriting this file.